assets/css/style.css

```diff
@@ -31,7 +31,7 @@
     display: flex;
     flex-direction: column;
     justify-content: flex-start;
-    width: 30%;
```

Comment:
I actually fixed this issue on my end as well. width display isn't necessary. There are fixed widths on elements that will set the size of the box.